connectivity/connectivity-demos-test/src_test/rest/bpm/TestCallFileUpload.java

```diff
@@ -0,0 +1,30 @@
+package rest.bpm;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import com.axonivy.connectivity.Data;
+
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.ExecutionResult;
+import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+
+@IvyProcessTest
+public class TestCallFileUpload
+{
+
+  @Test
+  @Disabled("there is a bug inside the ivyScript File implementation. The Process Test does not setup the basePath correctly...")
```

Comment:
ok as a first step - However I think the best thing would be to extract that 'pmvRequest IvyScriptingObjectContext' impl that focuses on IApplication into an independent class ... then we could re-use it here.
If that's too much work ... let's create a story for it. In the end also our users should see at least by example how to setup the file context correctly. Even better would be to make it work by default in the ivy-core

Reply:
https://jira.axonivy.com/jira/browse/XIVY-4592